RomM/api.py

```diff
@@ -496,7 +502,7 @@ def download_rom(self) -> None:
             self._status.downloading_rom = rom
             self._status.downloading_rom_position = i + 1
             dest_path = os.path.join(
-                self._file_system.get_sd1_platforms_storage_path(rom.platform_slug),
+                self._file_system._get_sd1_platforms_storage_path(rom.platform_slug),
```

Comment:
same with this one, `get_platforms_storage_path`, itll use the right one under the hood